frontend/packages/helm-plugin/src/components/forms/install-upgrade/HelmChartVersionDropdown.tsx

```diff
@@ -141,8 +150,9 @@ const HelmChartVersionDropdown: React.FunctionComponent<HelmChartVersionDropdown
 
     setFieldValue('chartVersion', value);
     setFieldValue('chartURL', chartURL);
-
-    coFetchJSON(`/api/helm/chart?url=${chartURL}`)
+    coFetchJSON(
+      `/api/helm/chart?url=${chartURL}&namespace=${namespace}&indexEntry=${chartIndexEntry}`,
+    )
```

Comment:
Same here, we need to escape at least `chartURL`. I would prefer if we escape all parameter values.